src/library-authoring/component-info/ComponentManagement.tsx

```diff
@@ -61,35 +74,57 @@ const ComponentManagement = () => {
       />
       {[true, 'true'].includes(getConfig().ENABLE_TAGGING_TAXONOMY_PAGES)
         && (
-        <Collapsible
-          defaultOpen
```

Comment:
Was it not possible to simply set `defaultOpen={sidebarComponentInfo?.collapse !== 'tags'}` ? That would be a lot simpler than making the collapsible into a controlled component.

Reply:
No, it only works when the component is first mounted.